### ISSUE TITLE
.abapgit.xml

### ISSUE THREAD

**Issue by larshp** [RainerWinkler/Moose-FAMIX-SAP-Extractor#40]
Hi Rainer,
Thanks for using abapGit

The `.abapgit.xml` file has to be moved to the root of the repository for abapGit to find it, and also 
`<STARTING_FOLDER>/</STARTING_FOLDER>`
needs to be changed to
`<STARTING_FOLDER>/src/</STARTING_FOLDER>`

If you are using an offline project to export the files, there is currently no way to edit the `.abapgit.xml` file before exporting. But this feature is planned https://github.com/larshp/abapGit/issues/630 and https://github.com/larshp/abapGit/issues/227

**Comment by RainerWinkler** [comment]
Hi Lars,




thank you! I changed it.




I cannot test it currently because I am still building the SAP system, I will use for the further development.




Best regards,

Rainer




PS: The files looked so clear to me, that I made some corrections in the file and not in the SAP system from which I exported. This should be OK, or?


-----Original-Nachricht-----
Von: "Lars Hvam" <notifications@github.com>
An: RainerWinkler/Moose-FAMIX-SAP-Extractor <Moose-FAMIX-SAP-Extractor@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Datum: 02/27/17 06:30
Betreff: [RainerWinkler/Moose-FAMIX-SAP-Extractor] .abapgit.xml (#40)


Hi Rainer,
Thanks for using abapGit

The .abapgit.xml file has to be moved to the root of the repository for abapGit to find it, and also
<STARTING_FOLDER>/</STARTING_FOLDER>
needs to be changed to
<STARTING_FOLDER>/src/</STARTING_FOLDER>

If you are using an offline project to export the files, there is currently no way to edit the .abapgit.xml file before exporting. But this feature is planned larshp/abapGit#630 and larshp/abapGit#227
